### PR TITLE
refactor(vsc): rebrand AAD to Microsoft Entra

### PR DESCRIPTION
**Pull request by tecton**
Related task: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/25519228

### CODE REVIEW

**Review by SLdragon — APPROVED** [review]

File: packages/vscode-extension/CHANGELOG.md [hunk 1]
```diff
@@ -540,12 +540,12 @@ Fixed:
 
 ## 1.2.2 - April 1 2021
 
-- Increase the timout limit when creating an AAD password for a bot registration.
+- Increase the timout limit when creating an Microsoft Entra password for a bot registration.
```
Comment (sffamily) [inline packages/vscode-extension/CHANGELOG.md:543]: an -> a

**Review by sffamily — COMMENTED** [review]
Question: for this update is this mandatory now? Changing it now will easily confuse people in short period time. We would delay this as long as it allows.

File: packages/vscode-extension/CHANGELOG.md (hunk 1 above)
Comment (tecton) [reply packages/vscode-extension/CHANGELOG.md]: Thanks, fixed.

**Comment by tecton** [comment]
> Question: for this update is this mandatory now? Changing it now will easily confuse people in short period time. We would delay this as long as it allows.

@sffamily this feature has P0 priority, and the due date is Nov 15th. I think in next release all our products including CLI and VS extension will be rebranded. If the renaming is confusing, shall we mention this change in docs or reference official post?

Here's the feature details: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/25518522.

**Review by sffamily — APPROVED** [review]

**Review by HuihuiWu-Microsoft — APPROVED** [review]

**Pull request merged**